thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -275,80 +278,85 @@ def init_model(self):
         model.to(dtype=torch.bfloat16)
         return model
 
+    def apply_thunder_transforms(self, model):
+        if "thunder" not in self.compile or self.distributed_mode == "none":
+            return model
+        if self.distributed_mode == "ddp":
+            from thunder.distributed import ddp
+
+            model = ddp(
+                model,
+                broadcast_from=0,
+                bucket_size_in_mb=self.ddp_bucket_size,
+            )
+            return model
+        elif self.distributed_mode == "fsdp":
+            from thunder.distributed import fsdp, FSDPType, FSDPBucketingStrategy
+
+            sharding_strategy = {"zero2": FSDPType.ZERO2, "zero3": FSDPType.ZERO3}[self.shard_mode]
+            bucketing_strategy = {
+                "none": FSDPBucketingStrategy.NONE,
+                "block": FSDPBucketingStrategy.BLOCK,
+                "layer": FSDPBucketingStrategy.LAYER,
+            }[self.bucketing_mode]
+            model = fsdp(
+                model,
+                broadcast_from=None,
+                sharding_strategy=sharding_strategy,
+                bucketing_strategy=bucketing_strategy,
+            )
+            return model
+        else:
+            raise ValueError(f"Invalid {self.distributed_mode=}")
+
     def setup_distributed(self, model):
-        if self.distributed_mode == "none":
+        if self.distributed_mode == "none" or "thunder" in self.compile:
```

Comment:
Not needed but just a thought - Since we are applying Thunder distributed transforms in a separate function, can we move the check to the `setup_distributed` call on line 244 just so that it is clear that `setup_distributed` doesn't do any Thunder-related things from just reading the init function calls? wdyt?